src/features/account/components/models/AddToUserCircleModal.tsx

```diff
@@ -0,0 +1,117 @@
+import { Center, Link, Text, VStack, useToast } from '@chakra-ui/react';
+import { Trans, t } from '@lingui/macro';
+import type { FunctionComponent } from 'react';
+import { useEffect, useMemo } from 'react';
+
+import { AvatarBlueIcon, RequestSignModal } from '../../../../components';
+import { useDispatch, useSelector } from '../../../../hooks';
+import {
+  SignatureErrorTypes,
+  useTypedSignTrustCredential,
+} from '../../../../hooks/useTypedSignTrustCredential';
+import type { ApplicationState } from '../../../../store';
+import { trimAddress } from '../../../../utils';
+import { addUserToUserCircle, setAddToUserModalOpen } from '../../store';
+
+type AddToUserCircleModalProps = {
+  subjectAddress: string;
+};
+
+export const AddToUserCircleModal: FunctionComponent<
+  AddToUserCircleModalProps
+> = ({ subjectAddress }) => {
+  const { addToUserModalOpen } = useSelector(
+    (state: ApplicationState) => state.accountProfile,
+  );
+  const dispatch = useDispatch();
+
+  const {
+    isLoading,
+    isVerified,
+    payload,
+    submitTypedSignRequest,
+    signatureError,
+  } = useTypedSignTrustCredential();
+
+  const toast = useToast({ position: 'top' });
```

Comment:
shall we have a file to manage all toast message, because they are very alike 

```
const useToastMsg = (toastOptions) => {
     const showErrorMsgA = (param) => 
        toast({
                  title: t`Failed to verify signature`,
                  description: signatureError.message,
                  status: 'error',
                  duration: 2000,
                  isClosable: true,
                });
         }
      return {
          showErrorMsgA
      }
}

```

Reply:
ok, it think this is a good point, let me create a custom hook for toast.